### PR TITLE
Added Cibyl to Compilers section

### PR DESCRIPTION
**Pull request by ecstaticcoder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,6 +320,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 
 ## Implementations/Compilers
  * [charly](https://github.com/charly-lang) - Charly Programming Language
+ * [cibyl](https://github.com/senselogic/CIBYL) - Lightweight language which compiles to Ruby and Crystal.
```
Comment (veelenga) [inline README.md]: Avoid a full stop (dot at the end) for consistency with other entries.

Comment (senselogic) [reply README.md]: Oops :$

**Comment by veelenga** [comment]
@ecstaticcoder why did you close it?

**Comment by senselogic** [comment]
The PR can't be integrated as is, sorry for that.

I need to change the explanation text to "lightweight **curly-bracket**
language which compiles to Ruby and Crystal", which is much clearer I think.

I'll fix the PR this evening :)

**Pull request merged**